lang/java/java.talon

```diff
@@ -44,6 +48,8 @@ op array:
 op new:
     insert("new ")
 
+^annotate with$: insert("@")
```

Comment:
This doesn't need to be anchored, it probably would work better unanchored.

Reply:
What is the difference, just curious? I don't really understand how it works

Reply:
The '^' prefix means the command has to be at the start of an utterance. I.e. '^my command: "hello "' means I can say 'my command word world' and get 'hello world' written out, but I can't say 'word world my command' (I'll get 'world' out, and then maybe a misrecognition or nothing).

The '$' suffix means the rest of the utterance after the command is discarded. So if I had 'my command$: "hello "' and said "my command word world ten dot" I'd just get 'hello ' out.

Generally you try not to use anchors since it prevents people from fluidly making utterances as long as they want. For example when editing code I'm often able to queue up a bunch of edits to one line then move to a different line and start editing there in one utterance. The main limiting factor is the potential misrecognitions from Talon requiring backtracking.

As implemented this command is the same as saying 'at sign'. Do you think something like 'annotate override' -> '@Overrides', 'annotate non null' -> '@NonNull' etc. might be a more useful pattern?

Reply:
Should either unanchor or remove this.